conf.py

```diff
@@ -56,12 +56,9 @@
 # Don't execute the forced photometry notebook until we base the CI on
 # the actual fornax image instead of the fresh installs
 # (aka tractor install pain).
+# Also blocked by https://github.com/nasa-fornax/fornax-demo-notebooks/issues/311 (Spitzer COSMOS).
 nb_execution_excludepatterns += ['multiband_photometry.md',]
 
-# We use the non-public IRSA bucket for ZTF data, cannot execute the generator
-# notebook until https://github.com/nasa-fornax/fornax-demo-notebooks/issues/311 is addressed
-nb_execution_excludepatterns += ['light_curve_generator.md',]
-
```

Comment:
@bsipocz is this the right thing to do here?

Reply:
yes, this looks good to me, once we move to the ODR bucket the notebook is not blocked any more with the bucket issue. (I may have other, unrelated issues though, so whether you can do this in the same PR or not will depend)

Reply:
Latest testing still failed with a dead kernel. Need to investigate why that is happening.

I suspect this is unrelated, but we probably need to exclude this from Windows testing due to the use of `multiprocessing`. I added a block to conf.py for that. I think this notebook should still work on Mac but need to check.